src/sharedServices/getDownloadAvailabilityResult.ts

```diff
@@ -11,7 +13,12 @@ export const getDownloadAvailabilityResult = async (
   }
 
   return {
-    hasAvailableDownload: record.downloadsRemaining > 0,
+    hasAvailableDownload:
```

Comment:
Now this is also checking time it might be nice to give it a more generic name like `canDownload`.  It might also be cleaner to calculate the value before we get to the return block.

Reply:
See this has been done now, so resolving this comment